stable/selenium/templates/firefoxDebug-deployment.yaml

```diff
@@ -84,5 +84,17 @@ spec:
 {{ toYaml .Values.firefoxDebug.nodeSelector | indent 8  }}
 {{- else if .Values.global.nodeSelector }}
 {{ toYaml .Values.global.nodeSelector | indent 8  }}
+{{- end }}
+      affinity:
+{{- if .Values.chromeDebug.affinity }}
```

Comment:
I think you meant .Values.firefoxDebug.affinity instead of .Values.chromeDebug.affinity (same typo below) ;-)

Reply:
thanks for review, it's fixed.